tests/btp/hack/registry-availibility-check.sh

```diff
@@ -10,6 +10,7 @@ echo $PASSWORD
 
 echo Testing Docker Registry availibility at: $REGISTRY_URL
 
+sleep 10
```

Comment:
The `sleep` is not needed here. I would suggest replacing `set -e` with the `set +e` and checking if curl does not return RESPONSE_CODE other than 200 or does not return an exit code other than 0.

If I'm not wrong this `sleep 10` is a way not to face situations when a curl returns with an exception and it's impossible to check the returned value because `set -e` prevents the rest of the script from running.

Reply:
Yes, thanks for that, it was a leftover when trying to run it on GCP. Now on Azure it is running flawless